package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@newrelic/apollo-server-plugin",
+  "version": "0.1.0",
+  "description": "Apollo Server plugin that adds New Relic Node.js agent instrumentation.",
+  "main": "/src/index.js",
+  "scripts": {
+    "lint": "eslint *.js lib tests",
+    "test": "npm run unit",
+    "unit": "tap --test-regex='(\\/|^tests\\/unit\\/.*\\.test\\.js)$' --no-coverage"
+  },
+  "files": [
+    "index.js",
+    "lib/*.js"
+  ],
```

Comment:
@carlonewrelic added 'files' declaration which will limit files that get packaged. adding an example of the output to the main PR description. we shouldn't need an npmignore at any point on this repo... we can tweak this allow list, as needed.

Reply:
Awesome!  💯